registrations/utils.py

```diff
@@ -53,4 +53,7 @@ def send_mass_html_mail(
 
 
 def get_email_noreply_address():
-    return "noreply@%s" % Site.objects.get_current().domain
+    return (
+        getattr(settings, "NOREPLY_ADDRESS", None)
```

Comment:
No need to have getattr here since the setting exists.

Reply:
@charn This is now changed in https://github.com/City-of-Helsinki/linkedevents/pull/684/commits/fb464f1fa591da600215be5f39665142999e6801.